book/src/concepts/build_rs.md

```diff
@@ -0,0 +1,33 @@
+<!--
+SPDX-FileCopyrightText: 2021 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+
+SPDX-License-Identifier: MIT OR Apache-2.0
+-->
+
+# `build.rs`
+
+We need to specify a `build.rs` so that we can parse the macros and generate relevant C++ code.
```

Comment:
I don't think it's a great idea to keep this content. It's redundant with Getting Started chapter. IMO it's a maintenance burden that doesn't add value.

Reply:
Removed, i've merged them all to a "Build Systems" concept page that links to the getting started sections and mentions how other build systems might be possible too.